docs/apache-airflow/howto/set-up-database.rst

```diff
@@ -167,6 +167,7 @@ In the example below, a database ``airflow_db`` and user  with username ``airflo
    CREATE USER airflow_user WITH PASSWORD 'airflow_pass';
    GRANT ALL PRIVILEGES ON DATABASE airflow_db TO airflow_user;
    -- PostgreSQL 15 requires additional privileges:
+   \c airflow_db;
```

Comment:
This isn't SQL statement

Reply:
meta-command is acceptable in psql 
reference: https://www.postgresql.org/docs/current/app-psql.html

Reply:
another extra reference:
- https://stackoverflow.com/questions/55590601/grant-privileges-to-a-specific-table-with-postgresql-relation-does-not-exis

Reply:
psql is a cli tool to interact with Postgres where you can run and do more things than query. The code block here is pure SQL statements.

Reply:
Meta-commands aren't SQL :)

Reply:
If we merge this, users who copy paste their code to their sql tools / automation -- will have failures